### DISCUSSION TITLE
高分屏界面出现两个分层

### DISCUSSION THREAD

**Discussion by yubius** [Richasy/Bili.Uwp#699] [问答]
电脑界面是出现两个分层，小米客服说是Windows适配高分屏的原因。这是软件原因还是硬件？
![微信图片_20211211075325](https://user-images.githubusercontent.com/79253940/145657166-0e8a7e42-23f1-4413-a90d-e057cae58b95.jpg)
![微信图片_20211211075329](https://user-images.githubusercontent.com/79253940/145657170-d70101ad-4242-4165-b424-ab47bb2b5d30.jpg)

**Comment by hez2010** [comment]
没有遇到这种问题，可以试试更新显卡驱动
